sigmastate/src/main/scala/sigmastate/eval/TreeBuilding.scala

```diff
@@ -380,6 +380,9 @@ trait TreeBuilding extends RuntimeCosting { IR: IRContext =>
       case GM.exp(In(obj), In(arg)) =>
         mkExponentiate(obj.asGroupElement, arg.asBigInt)
 
+      case GM.multiply(In(obj), In(arg)) =>
```

Comment:
How's that is not a consensus-breaking change?

Reply:
TreeBuilding is part of front-end compiler and is used to produce ErgoTree from ErgoScript. 
I was pretty sure about it, but checked again, it is never used as part of `verify` and `prove`.
This fixes a bug I found in front-end compiler.